src/UserSpaceInstrumentation/CMakeLists.txt

```diff
@@ -33,6 +33,22 @@ target_link_libraries(UserSpaceInstrumentation PUBLIC
         OrbitBase
         CONAN_PKG::abseil)
 
+# Test Lib is merely used in UserSpaceInstrumentationTests below. The binary
```

Comment:
Change "Test Lib" to the name of the CMake library or to "This test lib".

Reply:
done